CMakeLists.txt

```diff
@@ -48,7 +48,8 @@ set(SOURCE_FILES
     src/services/node-instance-service.cpp
     src/services/node-executor-service.cpp
     src/scaler.cpp
-    src/executor.cpp)
+    src/executor.cpp
+    src/dagParser.cpp)
```

Comment:
Can you rename the `dagParser.cpp` file into `dag-parser.cpp`? This way, it is more consistent with the rest of the codebase.